examples/guestbook-go/_src/Dockerfile

```diff
@@ -12,11 +12,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM google/golang:latest
+FROM golang:latest
```

Comment:
Can you pin this while we're at it? Sorry I didn't catch this the first time, but we've been caught too many times trying to rebuild images in a pinch and not being able to because the base builder image drifted.